README.md

```diff
@@ -1,8 +1,14 @@
 The Moby Project
 ================
+[![ci](https://github.com/moby/moby/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/ci.yml)
```

Comment:
I would put CI related badges after `Latest release` and `License`. This is not a primary info imo. We should also add the test workflow:

```suggestion
[![ci workflow](https://github.com/moby/moby/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/ci.yml)
[![test workflow](https://github.com/moby/moby/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/test.yml)
```